Description: compatibility with glibc 2.34
 In glibc 2.34, PTHREAD_STACK_MIN is no longer a constant, but a runtime call
 to sysconf.  In practice, our minimum stack size is going to be less than
 the minimum ntp defines of 64k, so just skip this compile-time check.
Author: Steve Langasek <steve.langasek@ubuntu.com>
Last-Update: 2021-12-08

Index: ntp-4.2.8p15+dfsg/libntp/work_thread.c
===================================================================
--- ntp-4.2.8p15+dfsg.orig/libntp/work_thread.c
+++ ntp-4.2.8p15+dfsg/libntp/work_thread.c
@@ -41,12 +41,6 @@
 #ifndef THREAD_MINSTACKSIZE
 # define THREAD_MINSTACKSIZE	(64U * 1024)
 #endif
-#ifndef __sun
-#if defined(PTHREAD_STACK_MIN) && THREAD_MINSTACKSIZE < PTHREAD_STACK_MIN
-# undef THREAD_MINSTACKSIZE
-# define THREAD_MINSTACKSIZE PTHREAD_STACK_MIN
-#endif
-#endif
 
 #ifndef THREAD_MAXSTACKSIZE
 # define THREAD_MAXSTACKSIZE	(256U * 1024)
